src/google_cloud_debugger/google_cloud_debugger_lib/dbg_enum.h

```diff
@@ -52,8 +100,14 @@ class DbgEnum : public DbgClass {
   // the value of an enum.
   static const std::string kEnumValue;
 
+  // The integral type of the enum.
+  CorElementType enum_type_ = CorElementType::ELEMENT_TYPE_END;
```

Comment:
Same comment about ELEMENT_TYPE_END

Reply:
Is this the best option we have here?  Is there ever a chance we end up with an enum_type_ of ELEMENT_TYPE_END?

Reply:
I think so. We don't process or assign anything to `ELEMENT_TYPE_END` in our code. It is also the default value of the `CorElementType` enum.